docs/users.md

```diff
@@ -64,28 +70,26 @@ By default, each newly uploaded dataset can only be accessed by `Admins` and `Da
 | Create scripts (visible to everyone)            	| Yes   	| No              	| Yes          	| No          	|
 | Upload Datasets via UI                           	| Yes   	| Yes             	| Yes          	| No          	|
 | Set team access permissions upon dataset upload    	| Yes   	| Yes              	| Yes          	| No          	|
-| Get tasks again after cancelling an instance     	| Yes   	| No              	| Yes          	| No          	|
+| Get tasks again after canceling an instance     	| Yes   	| No              	| Yes          	| No          	|
 | Access to wK Statistics Menu  	| Yes   	| No              	| Yes          	| No          	|
 
 
-## Registering New Users
-
-New users can signup for webKnossos by clicking on the `Register Now` button in the navigation bar.
-In the registration form, they need to specify some basic information, and confirm acceptance of the privacy agreement.
-
-If there is just one organization in the webKnossos instance, they are assigned to that organization automatically but need to be activated by an administrator, compare below.
+## Sign Up & Invitations for New User Accounts
```

Comment:
I would add a section for switching organizations.

Reply:
Done